pkgs/by-name/re/refinery-cli/package.nix

```diff
@@ -15,10 +15,15 @@ rustPlatform.buildRustPackage rec {
   src = fetchCrate {
     pname = "refinery_cli";
     inherit version;
-    hash = "sha256-gHW+5WWzk1H2O5B2sWdl6QcOeUbNvbdZZBD10SmE1GA=";
+    hash = "sha256-8XYbPiF77r2gQM3rE4Pca3EXKOXjXBetJPkkL8Bf5dI=";
+    # The `time` crate doesn't build on Rust 1.80+
+    # https://github.com/NixOS/nixpkgs/issues/332957
+    postFetch = ''
+      patch $out/Cargo.lock ${./time-crate.patch}
+    '';
```

Comment:
cargoPatches might be a nicer way to achieve this.

Reply:
Done. `cargoPatches` expected a git patch instead of the `diffutils` format, so I had to re-generate it.